_source/_posts/2021-01-24-osgeo-glossary-updated.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: custom-post
+title:  "First release of OSGeo Glossary (and how to cite)!"
+date:   2021-01-24 11:30:00 +0800
+categories: update
+---
+
+A new release to celebrate the new year:
+the https://wiki.osgeo.org/wiki/Lexicon_Committee[OSGeo Lexicon Committee]
+has released its first set of terms!
+
+The OSGeo Glossary is now filled with terms and ready to use.
+These terms have been consolidated from OSGeo projects and
+especially the OSGeo Geo4All newsletter.
+
+We thank the editing team for compiling, editing and refining
+the terms (in alphabetical order):
+
```

Comment:
Our team worked efficiently through the entire process of compiling the terms, editing and collating them into a sheet, and then preparing them for Glossarist (the tool where the terms were published).